src/funding/credit/config.jsx

```diff
@@ -35,18 +37,34 @@ export function getCreditConfig(): FundingSourceConfig {
 
     Logo: ({ locale, logoColor }) => {
       if (locale.country === COUNTRY.DE) {
-        return enableLogoCDNExperiment(CreditLogo, { locale, logoColor });
+        return __WEB__ ? (
+          <CreditLogoExternalImage locale={locale} logoColor={logoColor} />
+        ) : (
+          <CreditLogoInlineSVG locale={locale} logoColor={logoColor} />
+        );
       }
 
       return (
         <Fragment>
-          {enableLogoCDNExperiment(PPLogo, { logoColor })}
+          {__WEB__ ? (
+            <PPLogoExternalImage logoColor={logoColor} />
+          ) : (
+            <PPLogoInlineSVG logoColor={logoColor} />
+          )}
           <Space />
           <span optional>
-            {enableLogoCDNExperiment(PayPalLogo, { logoColor })}
+            {__WEB__ ? (
+              <PayPalLogoExternalImage logoColor={logoColor} />
+            ) : (
+              <PayPalLogoInlineSVG logoColor={logoColor} />
+            )}
             <Space />
           </span>
-          {enableLogoCDNExperiment(CreditLogo, { locale, logoColor })}
+          {__WEB__ ? (
+            <CreditLogoExternalImage locale={locale} logoColor={logoColor} />
+          ) : (
+            <CreditLogoInlineSVG locale={locale} logoColor={logoColor} />
+          )}
         </Fragment>
```

Comment:
should we do something like this? so we only need to check `__WEB__` once
{__WEB__ ? (
<Fragment>
  <PPLogoExternalImage logoColor={logoColor} />
  <Space />
  <span optional>
    <PayPalLogoExternalImage logoColor={logoColor} />
    <Space />
  </span>
  <CreditLogoExternalImage locale={locale} logoColor={logoColor} />
</Fragment>
) : (
<Fragment>
  <PPLogoInlineSVG logoColor={logoColor} />
  <Space />
  <span optional>
    <PayPalLogoInlineSVG logoColor={logoColor} />
    <Space />
  </span>
  <CreditLogoInlineSVG locale={locale} logoColor={logoColor} />
</Fragment>
)
}

Reply:
Good call!  Refactored.